pkgs/applications/science/electronics/kicad/versions.nix

```diff
@@ -3,49 +3,49 @@
 {
   "kicad" = {
     kicadVersion = {
-      version =			"5.1.8";
+      version =			"5.1.6";
```

Comment:
Is this a downgrade?

Reply:
Whoops! Rebase failure. Thanks for noticing.